lib/cocoapods/command/repo/push.rb

```diff
@@ -39,6 +41,7 @@ def initialize(argv)
           @podspec = argv.shift_argument
           @use_frameworks = !argv.flag?('use-libraries')
           @private = argv.flag?('private', true)
+          @message = argv.flag?('message')
```

Comment:
Why not just take this as an option so you don't need to open the external editor?